docs/how-to-guides/run-chainhook-as-a-service-using-stacks.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Run Chainhook as a Service using Stacks
+---
+
+# Run Chainhook as a Service using Stacks
+
+This document helps you configure the Stacks and chainhook nodes to extract on-chain data based on the predicate definition.
```

Comment:
```suggestion
This document helps you configure the Stacks and Chainhook nodes to extract on-chain data based on the predicate definition.
```

Reply:
This statement is modified to "You can run Chainhook as a service to evaluate Stacks blocks against your predicates. You can also dynamically register new predicates by enabling predicates registration API."